api/db-pouch.js

```diff
@@ -0,0 +1,10 @@
+const PouchDB = require('pouchdb-core');
+PouchDB.plugin(require('pouchdb-adapter-http'));
+PouchDB.plugin(require('pouchdb-mapreduce'));
+
+const { COUCH_URL } = process.env;
+// strip trailing slash from to prevent bugs in path matching
+const couchUrl = COUCH_URL && COUCH_URL.replace(/\/$/, '');
+const DB = new PouchDB(couchUrl);
+
+module.exports.medic = DB;
```

Comment:
Figured this might come in handy if we're planning to migrate from nano to pouch?